### ISSUE TITLE
Adding starting and ending date and simple gantt support

### ISSUE THREAD

**Issue by laoshaw** [go-gitea/gitea#7831]
this will make gitea more useful in practice than github, gitlab and redmine, simple due date and gantt support can make gitea extremely useful for simple agile teams too, so you don't need a huge jira for that.

**Comment by jag3773** [comment]
I agree, note the related discussion and comments at https://github.com/go-gitea/gitea/issues/3476#issuecomment-516517591
